src/core/variables.jl

```diff
@@ -52,6 +54,10 @@ struct EnergyVariableUp <: VariableType end
 
 struct EnergyVariableDown <: VariableType end
 
+struct EnergySlackUp <: VariableType end
```

Comment:
The way this is implemented, the energy targets are not slacks they are more like a surplus or a deficit. We should use better terminology to not imply these are equivalent to the slacks added via kwargs